src/sigstore.test.ts

```diff
@@ -117,13 +195,14 @@ describe('signAttestation', () => {
   it('returns the correct envelope', async () => {
     const sig = await signAttestation(payload, payloadType);
 
-    expect(sig).toEqual(bundle);
+    expect(sig).toEqual(Bundle.toJSON(bundle));
   });
 });
 
 describe('#verify', () => {
   const bundle: SerializedBundle = {
     mediaType: 'application/vnd.dev.sigstore.bundle+json;version=0.1',
+    dsseEnvelope: undefined,
     messageSignature: {
       messageDigest: {
         algorithm: 'SHA2_256',
```

Comment:
This is the one bit of the `OneOf` type that I don't like . . . the unused alternatives still need to be present, but set to `undefined`. However, this should really only come up in situations like this where we're manually constructing one of these objects to use in a test.